forge/forge/tvm_to_python.py

```diff
@@ -696,7 +696,14 @@ def populate_conv2d_transpose_args(graph, nid, compiler_cfg):
         )
     )
 
-    in_channel = next((n["attrs"]["shape"][0][0][0] for n in graph["nodes"] if n["name"] == "model.weight"), None)
+    in_channel = None
+    for input_ in node["inputs"]:
+        input_nid = input_[0]
+        input_node = graph["nodes"][input_nid]
+        if input_node["op"] == "parameter" and input_node["name"].endswith("weight"):
+            in_channel = input_node["attrs"]["shape"][0][0][0]
```

Comment:
Will this handle different shape ranks?

Reply:
Yes, inputs are either going to 3d or 4d for convtranspose2d which is handled here, I have added different shape rank shape inputs in sanity now.